src/components/pages/SavedJudges/SavedJudges.js

```diff
@@ -84,7 +82,6 @@ function SavedJudges({ savedJudges, deleteSavedJudge }) {
   savedJudges.forEach((item, idx) => {
     item.id = idx;
   }); // this is VERY hacky, but the table doesn't take data without ids
```

Comment:
Note to discuss: This was already in the code from before, but I am wondering if we should rephrase this comment, maybe simply say "id needed for table data"